stage0/run.go

```diff
@@ -314,7 +314,7 @@ func preparedWithPrivateUsers(dir string) (string, error) {
 
 // Run mounts the right overlay filesystems and actually runs the prepared
 // pod by exec()ing the stage1 init inside the pod filesystem.
-func Run(cfg RunConfig, dir string) {
+func Run(cfg RunConfig, dir string, dataDir string) {
```

Comment:
not thrilled about just passing the datadir in, seems a bit ad hoc... wondering if there's a better way for the main and stage0 packages to interface here.